### DISCUSSION TITLE
for @DIGITALCRIMINAL about newest commit

### DISCUSSION THREAD

**Discussion by J4k-The-Reaper** [UltimaHoarder/UltimaScraper#1303] [General]
@DIGITALCRIMINAL 
did you actually make the script now tell us the total download size (with progress bar if you use Windows Terminal only) instead of it downloading and counting the size as it downloads? that is really helpful and dope.

**Comment by UltimaHoarder** [comment]
Yes, since we're doing async requests, the script is able to get all header information and add up the total size of the files before downloading 😌

**Reply by J4k-The-Reaper** [reply]
i love the feature. and it seems much faster as well. helps me figure out storage space on hard drive before downloading. i absolutely love what you do. keep up the dope and amazing work!!
